source/tests/system/nisync_driver_api_tests.cpp

```diff
@@ -249,6 +249,23 @@ class NiSyncDriverApiTest : public ::testing::Test {
     return grpcStatus;
   }
 
+  ::grpc::Status call_MeasureFrequencyEx(ViConstString srcTerminal, ViReal64 duration, ViUInt32 decimationCount, ViReal64* actualDurationOut, ViReal64* frequencyOut, ViReal64* frequencyErrorOut, ViStatus* viStatusOut)
```

Comment:
This line is a bit long, put parameters on separate lines?

Reply:
Fixed in commit 5 and 6